packages/js/github-actions/actions/branch-label/action.yml

```diff
@@ -0,0 +1,45 @@
+name: Branch Type to Label
+description: Set PR labels according to the branch name.
+
+runs:
+  using: composite
+  steps:
+    - uses: actions/checkout@v2
```

Comment:
This step could be omitted since `actions-ecosystem/action-add-labels` can be run without the codebase.